README.md

```diff
@@ -137,7 +137,7 @@ docker build --tag peridio/peridiod .
 Running the container:
 
 ```bash
-podman run -it --rm --env PERIDIO_CERTIFICATE="$(cat /path/to/end-entity-certificate.pem)" --env PERIDIO_PRIVATE_KEY="$(cat /path/to/end-entity-private-key.pem)" peridio/peridiod:latest
+podman run -it --rm --env PERIDIO_CERTIFICATE="$(cat /path/to/end-entity-certificate.pem)" --env PERIDIO_PRIVATE_KEY="$(cat /path/to/end-entity-private-key.pem)" --cap-add=NET_ADMIN peridio/peridiod:latest
```

Comment:
What action requires `--cap-add=NET_ADMIN`? We probably should outline that in the readme. If I was seeing this command and unaffiliated I would be skeptical of the justification not being specified.

Reply:
Better yet would be not even requiring it, but if we must.

Reply:
remote access tunnels and the use of ip tables / wg-tools require this capability in order to modify the network table.